docs/Commiting.md

```diff
@@ -0,0 +1,30 @@
+# Commits
+
+This project follows the [conventional commits](https://www.conventionalcommits.org) guidelines to automate our version management and changelog generation.
+A pre-commit hook runs [commit lint](https://commitlint.js.org/) before every commit to ensure commit consistency.
+If you need some help commiting, try running `yarn commit` to commit using the [commitizen](https://github.com/commitizen/cz-cli) tool.
+
+> Be sure to check out those links for more information on how conventional commits guidelines and tools
+
+## Commit requirements
+
+- A commit message should follow the conventional commits guidelines
+- A commit message should include an issue number (DSY-*)
+
+### Commit Examples
+
+```sh
+docs: update readme, move contributing section to new file
+DSY-778
```

Comment:
We should add a example for external users, referencing a github issue.

Reply:
Não adicionei um exemplo porque hj eu dependo disso, tem que ajustar uma coisita no codigo pra aceitar esse padrão tbm. adicionei uma nota por enquanto e vou criar um card no jira so pra gente deixar o repo e processos amigaveis pra contribuições externas ;)

done in c5e361b